step/src/main/webapp/profile.jsp

```diff
@@ -88,12 +88,16 @@ limitations under the License.
     <div class="container">
         <div class="sub-container" id="list-selection">
             <h2> Your Profile </h2>
-            <div id="profile-pic"> <img src="avatar.png" alt="Profile Picture"> </div>
```

Comment:
Is this image still displayed when the user hasn't uploaded a profile picture?

Reply:
Yes, it's set in Javascript where if Datastore doesn't return a url for the user's image then it's set to avatar.png.